docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -407,7 +407,7 @@ $ gcloud compute instances create {INSTANCE_NAME} \
   --image-project confidential-space-images \
   --image-family confidential-space \
   --service-account {SERVICE_ACCOUNT} \
-  --metadata ^~^tee-image-reference={OPERATOR_IMAGE}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=prod~tee-env-API_TOKEN_SECRET_NAME={OPERATOR_KEY_SECRET_NAME}
+  --metadata ^~^tee-image-reference={OPERATOR_IMAGE}~tee-restart-policy=Never~tee-env-DEPLOYMENT_ENVIRONMENT=prod~tee-env-API_TOKEN_SECRET_NAME={OPERATOR_KEY_SECRET_FULL_NAME}
```

Comment:
@thomasm-ttd: looks great but we need the same mod in the JA corresponding file: https://github.com/IABTechLab/uid2docs/blob/main/i18n/ja/docusaurus-plugin-content-docs/current/guides/operator-private-gcp-confidential-space.md line 410. Code is not translated.